test/merge-patient-lab-iips-mocha.js

```diff
@@ -0,0 +1,138 @@
+// merge-patient-lab-iips-mocha.js
+
+var chai = require('chai');
+var chaiAsPromised = require('chai-as-promised');
+chai.should();
+chai.use(chaiAsPromised);
+
+var _ = require('underscore');
+var noflo = require('noflo');
+var componentFactory = require('../components/merge-patient-lab-iips.js');
+
+describe('merge-patient-lab-iips', function() {
+    var component = createComponent(componentFactory);
+    it("should have rdfpf object on nodeInstance", function() {
+        return Promise.resolve(component).then(function(component){
+            return component.rdfpf;
+        }).should.be.an('object');
+    });
+    it("should have createVni function", function() {
+        return Promise.resolve(component).then(function(component){
+            return component.rdfpf.createVni;
+        }).should.be.a('function');
+    });
+    it("createVni should return an object", function() {
+        return Promise.resolve(component).then(function(component){
+            return component.rdfpf.createVni('');
```

Comment:
It's not needed for this Sprint.  David wants it for a future sprint where we have non-IIP input and start handling split.  It came up because we are working with the API that will need it.